src/clue.js

```diff
@@ -2,28 +2,208 @@
+    image: "https://images2.minutemediacdn.com/image/upload/c_crop,h_1124,w_2000,x_0,y_50/v1560965041/shape/mentalfloss/586055-hasbro.jpg?itok=x4QQWTxa",
+    color: "white",
+  },
+
+  {
+    firstName: "Victor",
+    lastName: "Plum",
+    occupation: "Designer",
+    age: 22,
+    description: "Billionaire video game designer",
+    image: "https://66.media.tumblr.com/ee7155882178f73b3781603f0908617c/tumblr_phhxc7EhPJ1w5fh03_540.jpg",
+    color: "purple",
+  },
+
+  {
+    firstName: "Kasandra",
+    lastName: "Scarlet",
+    occupation: "Actor",
+    age: 31,
+    description: "She is an A-list movie star with a dark past",
+    image: "https://www.radiotimes.com/uploads/images/Original/111967.jpg",
+    color: "red",
+  },
+
+  {
+    firstName: "Eleanor",
+    lastName: "Peacock",
+    occupation: "Socialité",
+    age: 36,
+    description: "She is from a wealthy family and uses her status and money to earn popularity",
+    image: "https://vignette.wikia.nocookie.net/cluedo/images/b/b0/2016C5.jpg/revision/latest?cb=20200927103538",
+    color: "blue",
+  },
+
+  {
+    firstName: "Jack",
+    lastName: "Mustard",
+    occupation: "Retired Football player",
+    age: 62,
+    description: "He is a former football player who tries to get by on his former glory",
+    image: "https://static.independent.co.uk/s3fs-public/thumbnails/image/2016/07/04/08/unspecified-3.jpg",
+    color: "yellow",
+  }
+]
+
 
 // Rooms Array
+const roomsArray = [
+  {
+    name: "Dining Room",
+  },
 
-const roomsArray = [];
+  {
+    name: "Conservatory",
+  },
 
-// Weapons Array
+  {
+    name: "Kitchen",
+  },
 
-const weaponsArray = [];
+  {
+    name: "Study",
+  },
 
+  {
+    name: "Library",
+  },
 
-// ITERATION 2
+  {
+    name: "Billiard Room",
+  },
 
-function selectRandom() {}
+  {
+    name: "Lounge",
+  },
 
-function pickMystery() {}
+  {
+    name: "Ballroom",
+  },
 
+  {
+    name: "Hall",
+  },
+
+  {
+    name: "Spa",
+  },
+
+  {
+    name: "Living Room",
+  },
+
+  {
+    name: "Observatory",
+  },
+
+  {
+    name: "Theater",
+  },
+
+  {
+    name: "Guest House",
+  },
+
+  {
+    name: "Patio",
+  }
+]
 
-// ITERATION 3
 
-function revealMystery() {}
 
+// Weapons Array
+const weaponsArray = [
+  {
+    name: 'rope',
+    weight: 10,
+  },
+
+  {
+    name: 'knife',
+    weight: 8,
+  },
+  {
+    name: 'candlestick',
+    weight: 2,
+  },
+  {
+    name: 'dumbbell',
+    weight: 30,
+  },
+  {
+    name: 'poison',
+    weight: 2,
+  },
+  {
+    name: 'axe',
+    weight: 15,
+  },
+  {
+    name: 'bat',
+    weight: 13,
+  },
+  {
+    name: 'trophy',
+    weight: 25,
+  },
+  {
+    name: 'pistol',
+    weight: 20,
+  }
+]
+
+// ITERATION 2
+
+function selectRandom(someArray) {
+  let random = someArray[Math.floor(Math.random() * someArray.length)]
+  return random
+}
+let suspectRandom = selectRandom(suspectsArray)
+let weaponRandom = selectRandom(weaponsArray)
```

Comment:
Hi Thu! All in all great job, but I would do this inside of the pickMystery function

Reply:
like this:
```
function pickMystery() {
  let mystery = {
    suspect: selectRandom(suspectsArray),
    weapon: selectRandom(weaponsArray),
    room: selectRandom(roomsArray)
  }
  return mystery
}
```